app/models/miq_event.rb

```diff
@@ -14,8 +14,9 @@ class MiqEvent < EventStream
     }
   }
 
-  SUPPORTED_POLICY_AND_ALERT_CLASSES = [Host, VmOrTemplate, Storage, EmsCluster, ResourcePool, MiqServer,
-                                        ExtManagementSystem, ContainerImage].freeze
+  SUPPORTED_POLICY_AND_ALERT_CLASSES = [Host, VmOrTemplate, Storage, EmsCluster, ResourcePool,
+                                        MiqServer, ExtManagementSystem,
+                                        ContainerReplicator, ContainerGroup, ContainerNode, ContainerImage].freeze
```

Comment:
There is, though they're raised in a artificial way.  [This is existing functionality, I didn't add ContainerImage here]

I'm not sure now if this const is needed for control policies / compliance policies / both;
in any case we need both for ContainerImage, eg. for the policies out of the box:
https://github.com/ManageIQ/manageiq/blob/master/db/fixtures/miq_policy_sets.yml
The events are defined here:
https://github.com/ManageIQ/manageiq/blob/e6b829e8afba40de3022e93c031bc80def6a4e9f/db/fixtures/miq_event_definitions.csv#L166-L170

- ContainerImage is not quite a "real entity" we get external events for (at least currently); it's created during refresh when parsing Container data, and it simply has `after_create :raise_creation_event` which [raises the `containerimage_created` event][1].

- `containerimage_scan_complete` is raised [from Scanning::Job][2]

[1]: https://github.com/ManageIQ/manageiq/blob/3f4cf7fb9c3de711ee8668558abe2080ee7fc301/app/models/container_image.rb#L75-L77
[2]: https://github.com/ManageIQ/manageiq/blob/e5314d769defc97fbbf27045b2396058c277530b/app/models/manageiq/providers/kubernetes/container_manager/scanning/job.rb#L172